lib/rouge/lexers/apple_script.rb

```diff
@@ -305,7 +305,6 @@ def self.studio_properties
 
       operators = %r(\b(#{self.operators.to_a.join('|')})\b)
       classes = %r(\b(as )(#{self.classes.to_a.join('|')})\b)
-      literals = %r(\b(#{self.literals.to_a.join('|')})\b)
```

Comment:
It seems like the mistake here must really be later in the file where there's no rule created that uses this variable. 

Could you add the following before `rule commands …`?

```ruby
rule literals, Name::Builtin
```